### PR TITLE
Add support for Swift Package Manager

### PR DESCRIPTION
**Pull request by quanshousio**
Since Firebase has now officially support Swift Package Manager, albeit still in beta status (https://github.com/firebase/firebase-ios-sdk/issues/3136), this pull request adds support for SPM to this library.

### CODE REVIEW

**Comment by FranDepascuali** [comment]
I can confirm this works for my project, thank you!

**Comment by kshivang** [comment]
Awesome work. This worked fine for me as well, since SPM support is still in beta, I think it is reasonable to add beta warning and make cocoa method as recommended for now, in README.md.

File: README.md [hunk 1]
```diff
@@ -1,25 +1,36 @@
...
+#### Swift Package Manager
+`CombineFirebase` is available through [Swift Package Manager](https://swift.org/package-manager/). To install it, simply add `CombineFirebase` to an existing Xcode project as a package dependency:
+
+1. From the **File** menu, select **Swift Packages > Add Package Dependency...**
+2. Enter https://github.com/rever-ai/CombineFirebase into the package repository URL text field.
+3. Xcode should choose updates package up to the next version option by default.
+
+#### CocoaPods
+`CombineFirebase` is available through [CocoaPods](https://cocoapods.org). To install
 it, simply add the following line to your Podfile:
```
Comment (kshivang) [inline README.md:31]: I think it is reasonable to add beta warning and make cocoapods method as recommended for now.

**Review by kshivang — CHANGES_REQUESTED** [review]
Awesome work. This worked fine, since SPM support is still in beta, I think it is reasonable to add beta warning and make cocoaPods method as recommended for now, in README.md. Please change that part, I will gladly merge your PR.

File: README.md (hunk 1 above)
Comment (quanshousio) [reply README.md]: I just updated `README.md` in my last commit (e191eea), and also other minor stuffs.

Comment (quanshousio) [reply README.md]: I agree. I just updated `README.md` in my last commit (e191eea) and also other minor stuffs.

**Review by kshivang — APPROVED** [review]
Great, really appreciate your consistency efforts.

**Pull request merged**